codegen/runner/runner.go

```diff
@@ -86,4 +88,9 @@ func main() {
 	fmt.Printf("Generating main.go for gateway \n")
 	err = gatewaySpec.GenerateMain()
 	checkError(err, "Failed to generate main.go.")
+
+	fmt.Printf("Generating middleware schemas for gateway \n")
+	err = gatewaySpec.GenerateMiddlewareSchemas()
```

Comment:
I wanted to generate the json schemas from the middleware.Options structs, but based on Zhenghui's feedback that we should allow external dependencies, I'm shifting the configuration format around. It will be up to the middleware author to generate the schema and include it in the gateway configs.